draft-yasskin-http-origin-signed-responses.md

```diff
@@ -897,10 +897,14 @@ able to make even one unauthorized signature.
 Certificates with this extension MUST be revoked if an unauthorized entity is
 able to make even one unauthorized signature.
 
-Conforming CAs MUST mark this extension as critical, and clients MUST NOT accept
-certificates with this extension in TLS connections (Section 4.4.2.2 of
-{{!I-D.ietf-tls-tls13}}).  This simplifies security analysis of this protocol
-and avoids encouraging server operators to put exchange-signing keys on servers
+Conforming CAs MUST NOT mark this extension as critical. This allows new clients
+with old operating systems to handle the extension.
+
+Clients MUST NOT accept certificates with this extension in TLS connections
+(Section 4.4.2.2 of {{!I-D.ietf-tls-tls13}}).  This discourages server operators
+from using the same certificate to sign both TLS connections and signed
+exchanges, which in turn simplifies security analysis of this protocol and
+avoids encouraging server operators to put exchange-signing keys on servers
```

Comment:
I think the motivation might be better detailed in security considerations.

The reason this is a bit tricky is that by virtue of non-critical, clients that don't recognize this extension WILL accept this extension in TLS connections. So we don't have a hard security boundary. Because of this motivation, it seems like it makes sense to be discussed in the security section, calling out that this is not a security boundary itself, but simply an ecosystem health thing.